aem/build.gradle.kts

```diff
@@ -9,20 +7,31 @@ apply(from = rootProject.file("gradle/common.gradle.kts"))
 
 aem {
     environment {
+        docker {
+            init {
+                ensureDir(
+                        "httpd/cache",
+                        "httpd/logs"
+                )
+            }
+            containers {
+                "httpd" {
+                    reload {
+                        cleanDir(
+                                "/opt/aem/dispatcher/cache/content/example/live",
+                                "/opt/aem/dispatcher/cache/content/example/demo"
+                        )
+                        ensureDir("/usr/local/apache2/logs")
+                        exec("/usr/local/apache2/bin/httpd -k restart")
+                    }
+                }
+            }
+        }
         distributions {
             download("http://download.macromedia.com/dispatcher/download/dispatcher-apache2.4-linux-x86_64-4.3.2.tar.gz").then {
-                copyArchiveFile(it, "**/dispatcher-apache*.so", distributionFile("mod_dispatcher.so"))
+                copyArchiveFile(it, "**/dispatcher-apache*.so", file("mod_dispatcher.so"))
             }
         }
```

Comment:
Same here, since it is directly connected to httpd, why not to embed this dist in httpd container conf section?

Reply:
but what about shared distribution files...